post-router.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const db = require('./data/db');
+const router = express.Router();
+
+
+router.post('/', (req, res) => {
```

Comment:
again, excellent job on your endpoint logic. I know this is pretty similar to what you did the previous day but just with the router concept thrown in.  These reps are important because making endpoints like this is the bread-and-butter of backend.